roles/openshift_repos/tasks/centos_repos.yml

```diff
@@ -19,6 +19,6 @@
     src: "{{ item }}"
     dest: "/etc/yum.repos.d/{{ (item | basename | splitext)[0] }}"
   with_first_found:
-    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:2] }}.repo.j2"
+    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:3] }}.repo.j2"
```

Comment:
This looks incorrect.

Code in master:
'3.1.2'.split('.') should result in ['3', '1', '2']
'3.10.2'.split('.') should result in ['3', '10']

Reply:
You are right, but in that case it should evaluate
''.join('3.11.2'.split('.')[0:2]) to "311"
instead of
''.join('3.11.2'.split('.'))[0:2] to "31"

Am I correct?

Reply:
Updated in next commit